polytracker/scripts/poly-link.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+import sys
+import subprocess
+
+args = [x for x in sys.argv[1:]]
+final = ["llvm-link", "--only-needed"] + args
+subprocess.check_call(final)
```

Comment:
PEP compliance (newline at end of file):
```suggestion
subprocess.check_call(final)

```